lib/samples/create_planar_and_geodetic_buffers/README.md

```diff
@@ -0,0 +1,39 @@
+# Create planar and geodetic buffers
+
+Create a buffer around a map point and display the results as a graphic.
+
+![Image of create planar and geodetic buffers](create_planar_and_geodetic_buffers.png)
+
+## Use case
+
+Creating buffers is a core concept in GIS proximity analysis that allows you to visualize and locate geographic features contained within a polygon. For example, suppose you wanted to visualize areas of your city where alcohol sales are prohibited because they are within 500 meters of a school. The first step in this proximity analysis would be to generate 500 meter buffer polygons around all schools in the city. Any such businesses you find inside one of the resulting polygons are violating the law.
+
+## How to use the sample
+
+1. Tap on the map.
+2. A planar and a geodetic buffer will be created at the tap location using the distance (miles) specified.
+3. Continue tapping to create additional buffers. Notice that buffers closer to the equator appear similar in size. As you move north or south from the equator, however, the geodetic polygons become much larger. Geodetic polygons are in fact a better representation of the true shape and size of the buffer.
+4. Tap Clear to remove all buffers and start again.
```

Comment:
Is it better to say 'Tap Clear button in the Settings panel....`, since there is no Clear button in the UI

Reply:
I'd prefer the Clear button to be on the same row as the Settings button rather than in the Settings.

Reply:
I moved the Clear button so now the README matches